### PR TITLE
Release v4.5.0

### PR DESCRIPTION
**Pull request by aholovko**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,15 @@ Sharpbrake Changelog
 
 ### master
 
+### [v4.5.0][v4.5.0] (January 12, 2018)
+
+* Updated NLog dependencies
```
Comment (kyrylo) [inline CHANGELOG.md]: Not sure why it would be interesting for library users (perhaps I'm wrong, though?)

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,15 @@ Sharpbrake Changelog
...
+  ([#82](https://github.com/airbrake/sharpbrake/pull/82))
+* Added internal logging
```
Comment (kyrylo) [inline CHANGELOG.md]: This is definitely not worth mentioning.

File: CHANGELOG.md (hunk 1 above)
Comment (aholovko) [reply CHANGELOG.md]: Probably, it's not worth to mention. Will remove...

File: CHANGELOG.md (hunk 2 above)
Comment (aholovko) [reply CHANGELOG.md]: I would like to disagree here. Internal logging (tracing) is quite important when you need to debug some unexpected behavior. This is definitely a feature. I am going to refer to that functionality in this issue https://github.com/airbrake/sharpbrake/issues/77.

Comment (kyrylo) [reply CHANGELOG.md]: Yes, but it's internal, that is, for the developers of this project. Imagine an Airbrake customer reading this. How's that helpful to them?

Comment (aholovko) [reply CHANGELOG.md]: It's not for the developer of the Sharpbrake itself - this is for developers who integrate Sharpbrake into their projects but something goes bad, e.g. notices are not sent - how to check what is going on here? They can enable tracing and try to figure out the issue (like missing some configuration options or thread-related issue, etc.).

Comment (kyrylo) [reply CHANGELOG.md]: Ok, this does seem to be documenting of the public API. No objections!

**Pull request merged**